terraform/environments/xhibit-portal/waf-load-balancer.tf

```diff
@@ -400,3 +425,83 @@ resource "aws_wafv2_web_acl_logging_configuration" "waf_logs" {
   log_destination_configs = ["${aws_s3_bucket.waf_logs.arn}"]
   resource_arn            = aws_wafv2_web_acl.waf_acl.arn
 }
+
+resource "aws_s3_bucket_policy" "waf_logs_policy" {
+  bucket = aws_s3_bucket.waf_logs.bucket
+  policy = data.aws_iam_policy_document.s3_bucket_waf_logs_policy.json
+}
+
+data "aws_iam_policy_document" "s3_bucket_waf_logs_policy" {
+
+  statement {
+    sid = "AWSLogDeliveryWrite"
+    actions = [
+      "s3:PutObject",
+    ]
+    effect = "Allow"
+    resources = [
+      "${aws_s3_bucket.waf_logs.arn}/AWSLogs/FIND_ACCOUNT_ID_VARIABLE/*"
```

Comment:
Can just use * instead of using the account id - we have a seperate bucket in each environment, which means the account will only be writing to its own bucket.